sphinx_kidsfirst_theme/layout.html

```diff
@@ -0,0 +1,5 @@
+{% extends "!sphinx_rtd_theme/layout.html" %}
+{% block linktags %}
+    {{ super() }}
+    <link rel="stylesheet" href="https://kids-first.github.io/kf-sphinx-docs-theme/static/css/kidsfirst.css" type="text/css" />
```

Comment:
You may need a `.nojekyll` file so that gh won't try to build with jekyll. Otherwise, it may not put your files where you expect.

Reply:
Ok, for now I did manually push up the `.nojekyll` file to `gh-pages` branch but once I add circleci, I'll include that in the process that deploys gh-pages.

Reply:
I'm not sure you need circle for this. You should be able to have gh-pages build straight from master, then link to the static file directly.

Reply:
Ok, I made another PR #2 for just the css. Could you rev that one first please? I want get that merged first so I can make sure everything still works in this one

Reply:
Github automatically statically hosts all of your files.
The gh-pages link will be identical to https://raw.githubusercontent.com/kids-first/kf-sphinx-docs-theme/master/sphinx_kidsfirst_theme/static/css/kidsfirst.css

Reply:
@fiendish That unfortunately doesn't work..

You are right GH does statically host all your files, BUT, it serves the files back all as plain text or with `Content-Type: text/plain; charset=utf-8` and according to [this](https://stackoverflow.com/questions/17341122/link-and-execute-external-javascript-file-hosted-on-github) article, GH also started using `X-Content-Type-Options: nosniff`  which makes modern browsers do strict MIME type checks.  

I initially tried just doing the simple thing in my conf.py file, but it didn't work bc of the above :(
```
# in conf.py

html_css_files = ['https://raw.githubusercontent.com/kids-first/kf-sphinx-docs-theme/master/sphinx_kidsfirst_theme/static/css/kidsfirst.css']
```
I did consider the other options they mentioned in the article, but I didn't like the idea of using a random CDN to serve the file

Reply:
you're right. I did the same in some other personal projects and just forgot.